aten/src/ATen/native/cpu/ReduceNormKernel.cpp

```diff
@@ -0,0 +1,138 @@
+#define TORCH_ASSERT_ONLY_METHOD_OPERATORS
```

Comment:
Nit. When moving things around, it's good to provide a diff of the two files, as otherwise it's difficult to figure out what's different between the two.

Reply:
Also don't just move things around for the sake of it. CUDA files are split up because they take much longer to compile so smaller files is better for parallelisation, whereas CPU files are larger to amoritize the cost of pre-processing.